io.github.btpf.alexandria.yml

```diff
@@ -0,0 +1,75 @@
+---
+id: io.github.btpf.alexandria
+command: "alexandria"
+rename-icon: "alexandria"
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+- org.freedesktop.Sdk.Extension.node16
+separate-locales: false
+finish-args:
+- "--socket=wayland"
+- "--socket=fallback-x11"
+- "--share=ipc"
+- "--share=network"
+- "--device=dri"
+- "--filesystem=/run/media"
+# --- Start Build Fix ---
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: webkit2gtk-4.0
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.38.6.tar.xz
+        sha256: 1c614c9589389db1a79ea9ba4293bbe8ac3ab0a2234cac700935fae0724ad48b
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: <a href="webkitgtk\-(2\.38\.\d+)\.tar\.xz">
```

Comment:
This restricts webkit2gtk to an outdated version. It receives quite the number of security fixes each release.

Use this https://github.com/flathub/com.amazon.Workspaces/blob/a72f7e2df80013c7a5ba0da5912161bf6d1e3140/com.amazon.Workspaces.yml#L94-L98

You need to add libjpegxl temporarily until the runtime is updated to 45. It's a build dep of webkitgtk >= 2.42.0 https://github.com/flathub/com.amazon.Workspaces/blob/a72f7e2df80013c7a5ba0da5912161bf6d1e3140/com.amazon.Workspaces.yml#L63-L72